dbt/models/reporting/schema.yml

```diff
@@ -1,32 +1,157 @@
 sources:
   - name: reporting
-    tags:
-      - load_auto
     tables:
       - name: ratio_stats
-        description: '{{ doc("table_ratio_stats") }}'
+        tests:
+          - expression_is_true:
+              name: ratio_stats_no_nulls
+              expression: >-
+                year IS NOT NULL
+                  AND triad IS NOT NULL
+                  AND geography_type IS NOT NULL
+                  AND property_group IS NOT NULL
+                  AND assessment_stage IS NOT NULL
+                  AND sale_year IS NOT NULL
+          - expression_is_true:
+              name: ratio_stats_metrics_are_sensible
+              expression: >-
+                cod >= 0
+                  AND prd >= 0
+                  AND prb BETWEEN -1 AND 1
+                  AND mki >= 0
+                  AND triad IS NOT NULL
+                  AND geography_type IS NOT NULL
+                  AND property_group IS NOT NULL
+                  AND assessment_stage IS NOT NULL
+                  AND sale_year IS NOT NULL
+          - expression_is_true:
+              name: ratio_stats_within_directionality
+              expression: >-
+                within_20_pct >= within_10_pct
+                  AND within_10_pct >= within_05_pct
 
 models:
-  - name: reporting.res_report_summary
-    description: '{{ doc("table_res_report_summary") }}'
-    config:
-      tags:
-        - daily
-
+  - name: res_report_summary
```

Comment:
**issue (blocking):** The name key here needs to be prefixed with the schema. Also, don't remove the tags and description.

Note the warnings in this workflow under "Build models": https://github.com/ccao-data/data-architecture/actions/runs/6869938167/job/18683780160?pr=218